src/LetterChanger.java

```diff
@@ -1,37 +1,44 @@
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
 
 class LetterChanger {
     private static Scanner scanner = new Scanner(System.in);
-    OperationsOnWords onWords = new OperationsOnWords();
+    private OperationsOnWords operations = new OperationsOnWords();
 
-    private final String commendToUpperCase = "upper";
-    private final String commendTolowerCase = "lower";
+    private final String COMMAND_TO_UPPER_CASE = "upper";
+    private final String COMMAND_TO_LOWER_CASE = "lower";
+
+    private List<String> words = new ArrayList<>();
 
     public void addAndPrintWords() {
         addWords(getNumberOfWords());
-        String change = "";
-        while (!change.equals(commendTolowerCase) && !change.equals(commendToUpperCase)) {
-            change = readChangeString();
-            if (!change.equals(commendTolowerCase) && !change.equals(commendToUpperCase)) {
+        String change = chooseOption();
```

Comment:
I would move this whole switch to a new method and invoke it here. It would make code much cleaner.

Reply:
I  moved switch and option to write string to chooseOption();